atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -64,7 +64,8 @@ def run(self):
         pullspec = image.copy()
         pullspec.registry = registry.uri  # the image on Crane
 
-        digests = get_manifest_digests(pullspec, registry.uri, self.insecure, self.secret)
+        digests = get_manifest_digests(pullspec, registry.uri, self.insecure, self.secret,
+                                       throw_error=False)
```

Comment:
Not sure what you mean. Couldn't you just `pass` if you catch `RuntimeError`, and have the same effect as passing through a `throw_error` parameter?